src/components/updatemenus/attributes.js

```diff
@@ -49,6 +49,7 @@ var buttonsAttrs = {
 
 module.exports = {
     _isLinkedToArray: 'updatemenu',
+    _arrayAttrRegexps: [/^updatemenus\[(0|[1-9][0-9]+)\]\.buttons/],
```

Comment:
Yes, we could... I just put it there because that's where `_isLinkedToArray` lives and I thought it would be good to keep them together. I don't have a strong opinion either way.